apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java

```diff
@@ -41,11 +35,13 @@
 @Slf4j
 @Singleton
 public class DexOrderProcessor {
+    private static final int ORDERS_SELECT_SIZE = 30;
```

Comment:
how about moving these things to Constants?

Reply:
Is not necessary, used only for one method here